src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.html

```diff
@@ -4,31 +4,27 @@
     <li class="breadcrumb-item active">Hosts</li>
   </ol>
 </nav>
-<table class="table table-bordered">
-  <thead>
-  <tr>
-    <th>
-      Hostname
-    </th>
-    <th>
-      Services
-    </th>
-    <th>
-      Version
-    </th>
-  </tr>
-  </thead>
-  <tbody>
-  <tr *ngFor="let host of hosts">
-    <td>
-      {{ host.hostname }}
-    </td>
-    <td>
-      {{ host.services | serviceList }}
-    </td>
-    <td>
-      {{ host.ceph_version | cephShortVersion }}
-    </td>
-  </tr>
-  </tbody>
-</table>
+<ngx-datatable class="bootstrap oadatatable"
```

Comment:
Please use our data table component instead of using the "ngx-datatable" directly. Extend the component if needed.

Reply:
This is currently not possible because i didn't get it working to use pipes in TS column config. It only works using HTML syntax. So this is a blocker to re-use our datatable.